File: Programlar/Python/FM suallari part5/1.py
print("size of chessboard:")
n = int(input())
m = int(input())
print("coordinate:")
i = int(input())
j = int(input())
c = int(input("black(1) or white(0)?\n"))
first=1 if c==0 else 0
if (i-1)%2 == (j-1)%2: first=c
if m%2==0: print("equals")
elif n%2==0: print("equals")
else:
    if first==1: print("black")
    else: print("white")

#sahmat meselesi. hansisa xanaya teyin edilmis renge gore qurulmus sahmat taxtasinda hansi rengli xana daha coxdur?

File: Programlar/C++/FM suallari part5/1.cpp
#include<iostream>
using namespace std;
int main(){
int n,m,i,j,c;
cout<<"size of chessboard: \n";
cin>>n>>m;
cout<<"coordinate: \n";
cin>>i>>j;
cout<<"black(1) or white(0) ? \n";
cin>>c;
int first= c==0?1:0;
if((i-1)%2 == (j-1)%2)first=c;
if(m%2==0) cout<<"equals";
else if(n%2==0) cout<<"equals";
else {
	if(first==1)printf("black");
	else printf("white");
}
return 0;
}

//sahmat meselesi. hansisa xanaya teyin edilmis renge gore qurulmus sahmat taxtasinda hansi rengli xana daha coxdur?
